### DISCUSSION TITLE
Select all objects less than defined LENGTH

### DISCUSSION THREAD

**Discussion by valdisj** [OpenOrienteering/mapper#2286] [Questions & Answers]
Hi, How would it be possible to select all objects in the OOM file which are less than some length (say, <10 meters)? Reason for asking - after generating template, there are a lot of smaller irrelevant artefacts as vector objects and I would like to delete them in bulk. I know in OCAD you can select all objects by their properties, like lenght. How this is possible to be done in OOM Mapper?

**Comment by dl3sdo** [comment]
Hi @valdisj,
your feature request is related to #2211. It is currently not implemented.

In general my idea is to use read-only tags created by Mapper in combination with the 'Find' mechanism.
Some of these tags would become part of .omap file (e.g., creating date, date of last change (see #2148), editor) whereas other tags would only exist at runtime (e.g., length, area, min. length violation, min. area violation).

**Comment by dl3sdo** [comment]
See #2341 (Feature: Search for dynamic object properties)
